app/src/main/java/org/jboss/hal/client/bootstrap/tasks/CheckForUpdate.java

```diff
@@ -43,6 +44,16 @@ public CheckForUpdate(Environment environment, EventBus eventBus) {
         this.eventBus = eventBus;
     }
 
+    private String getVersionFromResponse(String responseText) {
+        JSONObject data = new JSONObject(JsonUtils.safeEval(responseText));
+        String version = data.get("tag_name").toString();
+        version = version.substring(1, version.length() - 1);
+        if (version.contains("v")) {
```

Comment:
Maybe `version.startsWith("v")` is better here?

Reply:
sure, will change it